book/09-git-and-other-scms/sections/client-bzr.asc

```diff
@@ -128,7 +128,7 @@ $ git push origin master
 Git's remote-helpers framework has some limitations that apply.
 In particular, these commands don't work:
 
-* git push origin :branch-to-delete (Bazaar can't accept ref deletions in this way.)
+* git push origin :branch-to-delete (Bazaar can't accept ref deletions in this way)
 * git push origin old:new (it will push 'old')
```

Comment:
```suggestion
* git push origin old:new (it will push `old`)
```

Reply:
@Morganov, 
I've covered this same snippet of code in different two PR, sorry about that (I didn't notice until now). Here I'm agree with your suggestion, but can you take a look into [this](https://github.com/progit/progit2/pull/1444#discussion_r446492758) suggestion or wait for @ben's reply? So that I've to do changes at only one place.

Reply:
Let's wait for @ben's reply.